src/home-page/HomeMain.css

```diff
@@ -0,0 +1,44 @@
+.sm-categories {
+  padding: 2rem;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+.sm-card-img img {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+.sm-categ-card {
+  background-color: black;
+  border-radius: 10px;
+}
+.sm-card-details {
+  color: white;
+  padding: 1rem 1rem 0 1rem;
+}
+/* .btn-primary {
```

Comment:
if it not necessary then you can remove this

Reply:
done